src/SFA.DAS.AssessorService.Web.UnitTests/MockedObjects/MockedCertificateApiClient.cs

```diff
@@ -1,28 +1,23 @@
 ﻿using System;
-using System.Collections.Generic;
 using FizzWare.NBuilder;
-using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Moq;
 using Newtonsoft.Json;
 using RichardSzalay.MockHttp;
 using SFA.DAS.AssessorService.Application.Api.Client;
 using SFA.DAS.AssessorService.Application.Api.Client.Clients;
 using SFA.DAS.AssessorService.Domain.Entities;
-using SFA.DAS.AssessorService.Settings;
 
 namespace SFA.DAS.AssessorService.Web.UnitTests.MockedObjects
 {
     public class MockedCertificateApiClient
     {
         public static CertificateApiClient Setup(Certificate certificate, Mock<ILogger<CertificateApiClient>> apiClientLoggerMock)
         {
-            var webConfigMock = new Mock<IWebConfiguration>();
-            var hostMock = new Mock<IHostEnvironment>();
-            hostMock
-                .Setup(m => m.EnvironmentName)
-                .Returns(Environments.Development);
-            var tokenServiceMock = new TokenService(webConfigMock.Object, hostMock.Object, false);
+            var tokenServiceMock = new Mock<ITokenService>();
+            tokenServiceMock
+                .Setup(m => m.GetToken())
+                .Returns(string.Empty);
```

Comment:
Wouldn't it be better to mock the `IHostEnvironment`'s property like the old code was doing? If for whatever reason we want the development environment to use a token in the future, that would cause this new test to fail